tests/gtest/avifgainmaptest.cc

```diff
@@ -884,7 +886,6 @@ TEST(GainMapTest, ConvertMetadata) {
   metadata_double.alternateOffset[1] = 0.0;
   metadata_double.baseHdrHeadroom = 1.0;
   metadata_double.alternateHdrHeadroom = 10.0;
```

Comment:
The original code has `backwardDirection = AVIF_TRUE`. To match that, should we make sure `baseHdrHeadroom > alternateHdrHeadroom`?

Reply:
No, that combination (where base is SDR and altr is HDR, but the gainmap "goes in reverse") is no longer supported in the spec.